pkgs/development/libraries/jcal/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, autoreconfHook
+, readline
+, which
+}:
+
+stdenv.mkDerivation rec {
+  name = "jcal";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "fzerorubigd";
+    repo = "jcal";
+    rev = "v${version}";
+    sha256 = "0m3g3rf0ycv2dsfn9y2472fa3r0yla8pfqk6gq00nrscsc3pp4zf";
+  };
+
+  buildInputs = [ autoreconfHook readline ];
```

Comment:
Again my bad, `autoreconfHook` should be in `nativeBuildInputs`. `readline` should stay in `buildInputs` though.